### PR TITLE
Added Snalisys IK/PinYin/Stconvert, Ingester LanguageDetect/OpenNLP, …

### PR DESCRIPTION
**Pull request by aparo**
…Prometheus Exporter and Learning-To-Tank Plugins

### Description
This pull request will add some of my managed plugins:
* [IK Analysis for OpenSearch](https://github.com/aparo/opensearch-analysis-ik): The IK Analysis plugin integrates Lucene IK analyzer (http://code.google.com/p/ik-analyzer/) into OpenSearch, support customized dictionary. This is a port of [medcl's IK Analysis](https://github.com/medcl/elasticsearch-analysis-ik).
* [OpenNLP Ingest Processor](https://github.com/aparo/opensearch-ingest-opennlp): This processor is doing named/date/location/'whatever you have a model for' entity recognition and stores the output in the JSON before it is being stored. This is a port of [spinscale's ElasticSearch OpenNLP ingest plugin](https://github.com/spinscale/elasticsearch-ingest-opennlp).
* [Langdetect Ingest Processor](https://github.com/aparo/opensearch-ingest-langdetect): Ingest processor doing language detection for fields. Port of https://github.com/spinscale/elasticsearch-ingest-langdetect
* [Pinyin Analysis for OpenSearch](https://github.com/aparo/opensearch-analysis-pinyin): This Pinyin Analysis plugin is used to do conversion between Chinese characters and Pinyin. Port of https://github.com/medcl/elasticsearch-analysis-pinyin
* [Prometheus Exporter Plugin](https://github.com/aparo/opensearch-prometheus-exporter): This is a builtin exporter from OpenSearch to *Prometheus* based on the code of Vincent Van Hollebeke and others at https://github.com/vvanholl/elasticsearch-prometheus-exporter . It collects all relevant metrics and makes them available to Prometheus via the OpenSearch REST API. 
* [Learning to Rank Plugin](https://github.com/aparo/opensearch-learning-to-rank): This is a port of https://github.com/o19s/elasticsearch-learning-to-rank to work with OpenSearch. It's a rewrite of some parts to be able to work with OpenSearch.
* [STConvert Analysis](https://github.com/aparo/opensearch-analysis-stconvert): STConvert is analyzer that convert Chinese characters between Traditional and Simplified. Port of https://github.com/medcl/elasticsearch-analysis-stconvert
 
### Issues Resolved
New plugins.

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _community_projects/opensearch-prometheus-learning-to-rank.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: OpenSearch Prometheus Learning to Rank Plugin
```
Comment (stockholmux) [inline _community_projects/opensearch-prometheus-learning-to-rank.md:2]: **Prometheus** Learning to Rank Plugin?

File: _community_projects/opensearch-analysis-stconvert.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: STConvert Analysis for OpenSearch
+description: STConvert is analyzer that convert Chinese characters between Traditional and Simplified. This is a port of medcl's STConvert Analysis for ElasticSearch (https://github.com/medcl/elasticsearch-analysis-stconvert).
```
Comment (stockholmux) [inline _community_projects/opensearch-analysis-stconvert.md:3]: typo: 'that convert'-> 'that convert**s**'

**Review by stockholmux — CHANGES_REQUESTED** [review]
- DCO check is failing, so you'll need to sign off your commits (squash then commit with `-s`)
- A few small changes noted in the review

**Comment by stockholmux** [comment]
@elfisher Take a look at this PR. @aparo has included some URLs in the text for the community project descriptions here.

We currently don't support links/markdown in the fields and it might be cleaner if we did. Thoughts @elfisher (and @aparo)?

**Comment by elfisher** [comment]
> @elfisher Take a look at this PR. @aparo has included some URLs in the text for the community project descriptions here.
> 
> We currently don't support links/markdown in the fields and it might be cleaner if we did. Thoughts @elfisher (and @aparo)?

Agreed. That would be nice. I'll pull this local and post a screenshot so we can discuss here with examples.

**Comment by elfisher** [comment]
image
Attaching image.

@stockholmux and @aparo is the desire to highlight the URLs or do link text to a url (e.g. `[text](url)`)?

**Comment by stockholmux** [comment]
Right. I kind of hate seeing raw, unclickable URLs in web browser. It is a bit of a change though and having links can distract from the primary path - getting users to click on the projects, so there is that balance that should be considered.

From a perspective of the site, it's a relatively simple change. It does introduce a slightly larger burden of checking for correctness as you could pass in some HTML that would break the layout.

**Comment by aparo** [comment]
I'll remove the links. To simplify the life to everyone.

File: _community_projects/opensearch-prometheus-learning-to-rank.md (hunk 1 above)
Comment (aparo) [reply _community_projects/opensearch-prometheus-learning-to-rank.md:2]: Typo in copy, I'll fix it

**Comment by krisfreedain** [comment]
@aparo - it's been a while - wanted to see if the edits and DCO could be accomplished soon

**Comment by krisfreedain** [comment]
never heard back.

**Pull request closed without merging**